src/Materials/Textures/babylon.mirrorTexture.ts

```diff
@@ -90,6 +90,7 @@
             this.ignoreCameraViewport = true;
 
             this.onBeforeRenderObservable.add(() => {
+                this.gammaSpace = !scene.imageProcessingConfiguration.isEnabled || !scene.imageProcessingConfiguration.applyByPostProcess;

```

Comment:
Double check that this is the correct place to set this? Should there be similar logic added to the refractionTexture?

Reply:
Could it be done on creation and in the onUpdateParameters event of the imageProcessingConfiguration ???